firebase-firestore/src/main/java/com/google/firebase/firestore/FirebaseFirestore.java

```diff
@@ -293,6 +293,21 @@ public WriteBatch batch() {
     return new WriteBatch(this);
   }
 
+  /**
+   * Executes a batchFunction on a newly created {@link WriteBatch} and then commits all of the
+   * writes made by the batchFunction as a single atomic unit.
+   *
+   * @param batchFunction The function to execute within the batch context.
+   * @return A Task that will be resolved when the write finishes.
```

Comment:
This comment inconsistently refers to the batch. Above it's "batch context" but here it's "write finishes". 

How about: `@return A Task that will be resolved when the batch has been committed.`?

Reply:
Changed